### PR TITLE
Default care card heading level

### PR DESCRIPTION
**Pull request by the-nathan-smith**
I've added the ability to alter the default heading level for care cards. This is done with the addition of a new setting. If the setting is not present within a project, the default heading level stays the same (3), thus not affecting users who do not wish to change their default.

### CODE REVIEW

**Comment by the-nathan-smith** [comment]
@mikemonteith if you don't mind reviewing. Also do I need the migration file as it is just for the testapp or is this still needed? Thanks

**Comment by DomBaker** [comment]
Just one more thing needs doing - update CHANGELOG.md 👍

File: testapp/testapp/settings/base.py [hunk 1]
```diff
@@ -165,3 +165,7 @@
 
 # Default field for primary keys
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+
+# Default heading level for care cards
+# Uncomment the line below and change the value to what you want (min=2 max=6)
+# DEFAULT_CARE_CARD_HEADING_LEVEL = "3"
```
Comment (mikemonteith) [inline testapp/testapp/settings/base.py:171]: This should be `DEFAULT_CARE_CARD_HEADING_LEVEL = 3` without the quotes probably?

**Review by mikemonteith — CHANGES_REQUESTED** [review]
You've got two new migrations. Should just be one.

File: testapp/testapp/settings/base.py (hunk 1 above)
Comment (DomBaker) [reply testapp/testapp/settings/base.py:171]: I believe it has been converted to a str @mikemonteith

**Comment by mikemonteith** [comment]
> Just one more thing needs doing - update CHANGELOG.md 👍

You'll want to add some documentation under `./docs` too as well as a note in the CHANGELOG

File: testapp/testapp/settings/base.py (hunk 1 above)
Comment (the-nathan-smith) [reply testapp/testapp/settings/base.py:171]: It does get converted to a string but only for the help text.

Comment (the-nathan-smith) [reply testapp/testapp/settings/base.py]: Removed the quotes.

**Review by the-nathan-smith — COMMENTED** [review]
Resolved issues.

**Review by the-nathan-smith — COMMENTED** [review]
Removed unnecessary migrations.

**Review by mikemonteith — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## v1.0.1
```
Comment (mikemonteith) [inline CHANGELOG.md:3]: This should be v1.1.0 since it's adding a new feature

**Review by mikemonteith — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (the-nathan-smith) [reply CHANGELOG.md]: Updated change log.

**Review by DomBaker — APPROVED** [review]

**Review by mikemonteith — APPROVED** [review]

**Comment by mikemonteith** [comment]
I merged this on the command line because I squashed commits into 1 commit. I thought github would be clever enough to realise and close it as successful merge but it didn't 😢

**Pull request closed without merging**